### PR TITLE
Yamato integration

### PR DESCRIPTION
**Pull request by todi1856**
This PR introduces some basic tests, which validates that Android Gradle project builds successfully.

The tests are ran by Yamato, and should prevent PRs unless the tests are green

### CODE REVIEW

File: .yamato/upm-ci.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+test_all_trigger:
+  name: Tests Trigger
+  agent:
+    type: Unity::VM
+    image: package-ci/win10:stable
+    flavor: b1.large
+  commands:
+    - dir
+  triggers:
```
Comment (manugildev) [inline .yamato/upm-ci.yml:67]: Is it worth having a ~weekly trigger against trunk? Release cadence of 6000.0+ is weekly.

```yaml
  triggers:
    recurring:
      - branch: master
        frequency: weekly
```

Comment (todi1856) [reply .yamato/upm-ci.yml:67]: sounds reasonable

Comment (todi1856) [reply .yamato/upm-ci.yml:67]: https://github.com/Unity-Technologies/uaal-example/pull/106/commits/88a131b5f94603c2ea2c5d0c642fc6663af956ec thx !

Comment (todi1856) [reply .yamato/upm-ci.yml:67]: @manugildev since you commented, you need approve now :)

**Review by manugildev — APPROVED** [review]

**Review by VeryHappyYoung — APPROVED** [review]
Thank you!

**Review by Pavel-Unity — COMMENTED** [review]
LGTM

**Pull request merged**